test/chi/js/popover.pug

```diff
@@ -0,0 +1,67 @@
+---
+title: Popover
+---
+script(src='/js/chi.js')
+
+- var moreSides = ["top", "top-start", "top-end", "right", "right-start", "right-end", "bottom", "bottom-start", "bottom-end", "left", "left-start", "left-end"];
+
+// Method 1 - Data Target
+button#popover-1b.chi-button(data-target='#popover-in-html',data-cy='show-popover') Popover
+// Put the following code anywere in the DOM, e.g. at the end of the DOM
+section#popover-in-html.chi-popover(aria-modal='true', role='dialog', aria-label='Popover title')
+  header.chi-popover__header
+    h2.chi-popover__title Popover title
+  .chi-popover__content
+    p.chi-popover__text Popover content
+script.
```

Comment:
Let's consolidate `chi.popover()` inside one `script.` on the button of the pug file